cmd/run.go

```diff
@@ -138,6 +138,7 @@ File paths can be specified as URLs to resolve ambiguity in paths containing col
 	addConfigOverrides(runCommand.Flags(), &cmdParams.rt.ConfigOverrides)
 	addConfigOverrideFiles(runCommand.Flags(), &cmdParams.rt.ConfigOverrideFiles)
 	addBundleModeFlag(runCommand.Flags(), &cmdParams.rt.BundleMode, false)
+	runCommand.Flags().BoolVar(&cmdParams.rt.Incognito, "incognito", false, "enables privacy mode wherein OPA does not report version information to an external service")
```

Comment:
Let's make this more concise:

```
disables anonymous version reporting (see: https://openpolicyagent.org/docs/latest/privacy)
```

Reply:
I've been mulling this over for a while.. figured maybe we should discuss. How do you guys feel about potentially changing the flag to like `--skip-version-check` or something rather than `--incognito`? I guess I worry about the potential negative implications of running without `--incognito` and privacy when compared to the same named feature in chrome, or even if you lookup the definition of incognito its "having one's true identity concealed". The implication being that if you are running without this flag enabled your identity is _not_ being concealed, which sounds way worse then doing a version report.

Reply:
Incognito certainly comes with additional presumed semantics.

Reply:
We can change the flag name to something more generic like `--skip-reporting`, if in the future the client wants to report more that the version.

Reply:
I like skip-version-check better as version checking is the functionality provided for the user herein.

Reply:
`--incognito` changed to `--skip-version-check`. Help comment updated to `disables anonymous version reporting (see: https://openpolicyagent.org/docs/latest/privacy)`

Reply:
I like `--skip-version-check`. The `--incognito` flag was more relevant when the scope was broader.